src/App.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="container">
+    <nav>
+      <h1 class="appName">Project Geo</h1>
+      <button v-if="visible" @click.prevent="doLogout" class="logout">
+        Logout
+      </button>
+    </nav>
+    <main class="mainView">
+      <router-view />
+    </main>
+  </div>
+</template>
+<script>
+import { mapState } from "vuex";
+export default {
+  name: "MainComponents",
+  created() {
+    this.$store.dispatch("loadToken");
+  },
+  methods: {
+    doLogout() {
+      this.$store.dispatch("logout");
+    },
+  },
+  computed: {
+    ...mapState(["visible"]),
+  },
+};
+</script>
+<style lang="scss">
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@200&family=Montserrat:wght@200&display=swap");
+body {
+  font-family: "Inter", sans-serif;
+  // background-color: #67e0a3;
```

Comment:
@JatinThakur14 Dont need to change this now, but in future, generally remove commented code when you push commits. The code is anyway readable from git version history